pyproject.toml

```diff
@@ -16,16 +17,18 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
     "dask",
+    "dask[dataframe]",
+    "dask[distributed]",
```

Comment:
I believe these three lines should be collapsed into `dask[dataframe,distributed]`